acceptance_tests/features/steps/authentication.py

```diff
@@ -0,0 +1,32 @@
+from behave import given, then, when
+
+from acceptance_tests.features.pages import sign_in_respondent
+from acceptance_tests.features.pages import sign_in_internal
+from acceptance_tests import browser
+from config import Config
+
+
+@given('the respondent is signed into their account')
+def signed_in_frontstage(_):
```

Comment:
I don't think we should be using frontstage here, I think it should be respondent